.github/workflows/nightly-build.yml

```diff
@@ -20,8 +20,8 @@ jobs:
         run: |
           set -eux
           pip install twine
-          apt-get update
-          apt-get install sudo -y
+          sudo apt-get update
+          sudo apt-get install sudo -y
```

Comment:
You wouldn't need the line 24, sudo is already shipped with the github-actions ubuntu18.04 image.

Just running `sudo apt-get update` should suffice.

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#administrative-privileges-of-github-hosted-runners